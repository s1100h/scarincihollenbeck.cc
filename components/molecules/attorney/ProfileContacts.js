import Link from 'next/link';
import { BsFillEnvelopeFill } from 'react-icons/bs';
import { FaLinkedin } from 'react-icons/fa';
import { IoPrintSharp } from 'react-icons/io5';
import { MdLocalPhone, MdLocationPin } from 'react-icons/md';
import { AddressBox } from 'styles/attorney-page/AttorneyProfile.style';

const ProfileContacts = ({
  contact, offices, fax, linkedIn,
}) => {
  const { phoneNumber, email } = contact || {};

  return (
    <AddressBox>
      <p className="contacts-title">Contacts</p>

      <ul className="contacts-list">
        <li className="contacts-item">
          <MdLocationPin />
          {offices?.map((office, idx) => (
            <Link
              className="contacts-link"
              href={office.uri || office.link}
              key={office.id || office.ID}
            >
              {office.name}
              {idx !== offices.length - 1 && ', '}
            </Link>
          ))}
        </li>

        <li className="contacts-item">
          <a href={`tel:${phoneNumber}`} className="contacts-link">
            <MdLocalPhone />
            <span>{phoneNumber}</span>
          </a>
        </li>

        {fax && (
          <li className="contacts-item">
            <IoPrintSharp />
            {fax}
          </li>
        )}

        <li className="contacts-item">
          <a href={`mailto:${email}`} className="contacts-link">
            <BsFillEnvelopeFill />
            <span>{email}</span>
          </a>
        </li>

        <li className="contacts-item">
          <a
            href={linkedIn?.url}
            target="_blank"
            rel="noreferrer"
            className="contacts-link"
          >
            <FaLinkedin />
            <span>{linkedIn?.channel}</span>
          </a>
        </li>
      </ul>
    </AddressBox>
  );
};

export default ProfileContacts;
