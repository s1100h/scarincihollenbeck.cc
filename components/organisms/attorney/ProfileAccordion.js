import DisclaimerText from 'components/atoms/DisclaimerText';
import AccordionItem from 'components/molecules/attorney/AccordionItem';
import { Accordion } from 'react-bootstrap';
import {
  ProfileAccordionHolder,
  ProfileAccordionWrapper,
} from 'styles/attorney-page/ProfileAccordion.style';
import { ContainerDefault } from 'styles/Containers.style';
import empty from 'is-empty';
import Link from 'next/link';
import ProfileClients from 'components/molecules/attorney/ProfileClients';
import dynamic from 'next/dynamic';
import { StandardLightBlueButton } from 'styles/Buttons.style';
import AccordionDynamicItem from 'components/molecules/attorney/AccordionDynamicItem';
import GallerySlider from 'components/molecules/attorney/GallerySlider';
import MediaSlider from 'components/molecules/attorney/MediaSlider';
import { useRouter } from 'next/router';
import { reservedAccordionTitles } from 'utils/constants';
import { getPaginationData } from '../../../requests/getPaginationData';
import { postsForPaginationByAuthorIdQuery } from '../../../requests/graphql-queries';

const AwardsSlider = dynamic(() => import('components/molecules/home/AwardsSlider'));

const BlogsBox = dynamic(() => import('../../molecules/attorney/BlogsBox'));

const renderBlogPosts = (data, config, blogTitles, isWide, name) => blogTitles?.includes(config?.title) && (
<AccordionItem
  as="li"
  eventKey={`${config?.actionKey}-${name}`}
  title={config?.title}
>
  <BlogsBox
    queryParamsForPagination={config?.queryParams}
    paginationData={data}
    isWideCards={isWide}
  />
</AccordionItem>
);

const checkOnDuplicate = (title) => {
  if (reservedAccordionTitles.includes(title)) {
    return (
      <>
        {title}
        <span className="duplicate">Duplicate</span>
      </>
    );
  }

  return title;
};

const ProfileAccordion = ({
  clients,
  awards,
  attorneyBiography,
  additionalTabs,
  representativeMatters,
  gallery,
  mediaItems,
  presentationsItems,
  publicationsItems,
  videos,
  govLawPosts,
  blogTitles,
  name,
  authorId,
}) => {
  const router = useRouter();

  // News Press Releases section START
  const newsPressReleasesConfig = {
    title: 'News & Press Releases',
    queryParams: 'news-press-releases-page',
    actionKey: 'news-and-press',
    params: {
      authorId,
      categoryId: 98,
      currentPage: router?.query?.['news-press-releases-page'] || 1,
      itemsPerPage: 6,
    },
  };

  const newsPressReleasesPaginationData = getPaginationData(
    postsForPaginationByAuthorIdQuery,
    newsPressReleasesConfig.params,
  );
  // News Press Releases section END

  // Blog section START
  const blogConfig = {
    title: 'Blog',
    queryParams: 'blogs-page',
    actionKey: 'blogs',
    params: {
      authorId,
      categoryId: 599,
      currentPage: router?.query?.['blogs-page'] || 1,
      itemsPerPage: 3,
    },
  };

  const blogPostsPaginationData = getPaginationData(
    postsForPaginationByAuthorIdQuery,
    blogConfig.params,
  );
  // Blog section END

  // Events section START
  const eventsConfig = {
    title: 'Events',
    queryParams: 'events-page',
    actionKey: 'events',
    params: {
      authorId,
      categoryId: 99,
      currentPage: router?.query?.['events-page'] || 1,
      itemsPerPage: 6,
    },
  };

  const eventsPostsPaginationData = getPaginationData(
    postsForPaginationByAuthorIdQuery,
    eventsConfig.params,
  );
  // Events section END

  return (
    <ProfileAccordionWrapper data-testid="profile-accordion">
      <ContainerDefault>
        <ProfileAccordionHolder>
          <Accordion as="ul" alwaysOpen>
            {!empty(awards) && (
              <AccordionItem
                as="li"
                eventKey={`profile-awards-${name}`}
                title="Awards"
              >
                <StandardLightBlueButton as={Link} href="/awards">
                  Award Methodology
                </StandardLightBlueButton>
                <AwardsSlider items={awards} isLightVariant />
              </AccordionItem>
            )}

            {!empty(attorneyBiography?.biographyContent) && (
              <AccordionDynamicItem
                tag="li"
                eventKey={`profile-biography-${name}`}
                title="Full Biography"
                content={attorneyBiography?.biographyContent}
                isTwoColumns
              />
            )}

            {!empty(representativeMatters) && (
              <AccordionDynamicItem
                tag="li"
                eventKey={`profile-representative-matters-${name}`}
                title="Representative Matters"
                ulProps={{ $columnsCountUl: 2, $columnGapUl: 40 }}
                content={representativeMatters}
                disclaimer="* Results may vary depending on your particular facts and legal circumstances."
              />
            )}

            <ProfileClients clients={clients} name={name} />

            {renderBlogPosts(
              newsPressReleasesPaginationData,
              newsPressReleasesConfig,
              blogTitles,
              false,
              name,
            )}

            {renderBlogPosts(
              blogPostsPaginationData,
              blogConfig,
              blogTitles,
              false,
              name,
            )}

            {!empty(mediaItems) && (
              <AccordionItem
                as="li"
                eventKey={`profile-media-${name}`}
                title="Media"
              >
                <MediaSlider items={mediaItems} />
              </AccordionItem>
            )}

            {!empty(presentationsItems) && (
              <AccordionItem
                as="li"
                eventKey={`profile-presentations-${name}`}
                title="Presentations"
              >
                <MediaSlider items={presentationsItems} />
              </AccordionItem>
            )}

            {renderBlogPosts(
              eventsPostsPaginationData,
              eventsConfig,
              blogTitles,
              true,
              name,
            )}

            {!empty(videos) && (
              <AccordionItem
                as="li"
                eventKey={`profile-videos-${name}`}
                title="Video"
              >
                <MediaSlider items={videos} />
              </AccordionItem>
            )}

            {!empty(gallery) && (
              <AccordionItem
                as="li"
                eventKey={`profile-gallery-${name}`}
                title="Gallery"
              >
                <GallerySlider items={gallery} />
              </AccordionItem>
            )}

            {!empty(publicationsItems) && (
              <AccordionItem
                as="li"
                eventKey={`profile-publications-${name}`}
                title="Publications"
              >
                <MediaSlider items={publicationsItems} />
              </AccordionItem>
            )}

            {!empty(govLawPosts?.posts) && (
              <AccordionItem
                as="li"
                eventKey={`profile-gov-law-${name}`}
                title="Government & Law"
              >
                <MediaSlider items={govLawPosts?.posts} />
              </AccordionItem>
            )}

            {!empty(additionalTabs)
              && additionalTabs.map(
                (tab) => !empty(tab?.content) && (
                <AccordionDynamicItem
                  tag="li"
                  key={`${tab?.id}-additional-tab`}
                  eventKey={`profile-additional-${tab?.id}-${name}`}
                  title={checkOnDuplicate(tab?.title)}
                  content={tab?.content}
                />
                ),
              )}
          </Accordion>
          <DisclaimerText text="No Aspect of the advertisement has been approved by the Supreme Court. Results may vary depending on your particular facts and legal circumstances." />
        </ProfileAccordionHolder>
      </ContainerDefault>
    </ProfileAccordionWrapper>
  );
};

export default ProfileAccordion;
