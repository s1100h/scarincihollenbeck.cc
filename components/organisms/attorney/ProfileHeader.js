import React, { useState, useEffect } from 'react';
import ProfileContacts from 'components/molecules/attorney/ProfileContacts';
import ProfileImage from 'components/molecules/attorney/ProfileImage';
import {
  ProfileActions,
  ProfileBgImage,
  ProfileBio,
  ProfileBioListItems,
  ProfileBioText,
  ProfileBioTitle,
  ProfileButtons,
  ProfileDesignation,
  ProfileHeaderHolder,
  ProfileHeaderLeft,
  ProfileHeaderRight,
  ProfileHeaderSection,
  ProfileName,
  ProfileTitle,
} from 'styles/attorney-page/AttorneyProfile.style';
import { ContainerDefault } from 'styles/Containers.style';
import PDFIcon from 'components/common/icons/PDFIcon';
import BusinessCard from 'components/common/icons/BusinessCard';
import Link from 'next/link';
import { StandardBlueButton } from 'styles/Buttons.style';
import ModalWindow from 'components/common/ModalWindow';
import { FormBox } from 'styles/AboutAuthorFormCard.style';
import ContactForm from 'components/shared/ContactForm/ContactForm';
import ProfileServices from 'components/molecules/attorney/ProfileServices';
import { JSXWithDynamicLinks } from 'components/atoms/micro-templates/JSXWithDynamicLinks';
import empty from 'is-empty';
import WhiteButton from 'components/molecules/attorney/WhiteButton';
import ProfileBioList from 'components/molecules/attorney/ProfileBioList';
import PostBreadCrumbs from '../post/PostBreadcrumbs';

const useDesignationHook = (title) => {
  const [designation, setDesignation] = useState(title);
  useEffect(() => {
    if (title === 'Red Bank, NJ Managing Partner') {
      setDesignation('Red Bank, NJ  Office Managing Partner');
    }

    if (title === 'Managing Partner') {
      setDesignation('Firm Managing Partner');
    }

    if (title === 'Washington, D.C. Managing Partner') {
      setDesignation('Washington, D.C. Office Managing Partner');
    }
    setDesignation(title);
  }, [title]);

  return [designation, setDesignation];
};

const ProfileHeader = ({
  name,
  profileImage,
  title,
  offices,
  coChair: coChairs,
  chair: chairs,
  contact,
  representativeVideo,
  practices,
  attorneyBiography,
  education,
  barAdmissions,
  biography,
  affiliations,
  additionalInfo,
}) => {
  const [designation] = useDesignationHook(title);
  const [isContactModal, setIsContactModal] = useState(false);
  const linkedIn = contact?.socialMediaLinks?.filter(
    (a) => a.channel === 'LinkedIn',
  )[0];
  const profileImageProps = {
    name,
    profileImage,
    representativeVideo,
  };

  const profileDetailsProps = {
    offices,
    fax: contact?.fax,
    contact,
    linkedIn,
  };

  return (
    <ProfileHeaderSection data-testid="profile-header">
      <ContainerDefault>
        <PostBreadCrumbs />
        <ProfileHeaderHolder>
          <ProfileHeaderLeft>
            <ProfileImage {...profileImageProps} />
            <ProfileBgImage
              src="/images/profile-attorney-bg.webp"
              width={700}
              height={900}
              alt="Profile background"
              priority
              sizes="(max-width: 768px) 100vw, (max-width: 1680px) 480px, 700px"
              loading="eager"
            />

            <ProfileActions>
              {!empty(contact?.pdf) && !empty(contact?.vizibility) && (
                <ProfileButtons>
                  {!empty(contact?.pdf) && (
                    <WhiteButton
                      as={Link}
                      rel="noopener noreferrer"
                      target="_blank"
                      href={contact?.pdf}
                      text="Print Bio"
                      icon={<PDFIcon />}
                    />
                  )}
                  {!empty(contact?.vizibility) && (
                    <WhiteButton
                      as={Link}
                      rel="noopener noreferrer"
                      target="_blank"
                      href={contact?.vizibility}
                      text="Business Card"
                      icon={<BusinessCard />}
                    />
                  )}
                </ProfileButtons>
              )}

              <StandardBlueButton onClick={() => setIsContactModal(true)}>
                Contact now
              </StandardBlueButton>

              <ModalWindow
                isOpen={isContactModal}
                setOpenModal={setIsContactModal}
              >
                <FormBox>
                  <p className="contact-form-title">Let`s get in touch!</p>
                  <ContactForm blockName="profile-contact-form" />
                </FormBox>
              </ModalWindow>

              <ProfileContacts {...profileDetailsProps} />
            </ProfileActions>
          </ProfileHeaderLeft>

          <ProfileHeaderRight>
            <ProfileTitle>
              <ProfileName>{name}</ProfileName>
              <ProfileDesignation>{designation}</ProfileDesignation>
            </ProfileTitle>

            <ProfileServices
              coChairs={coChairs}
              chairs={chairs}
              services={practices}
            />

            {(!empty(attorneyBiography?.miniBio) || !empty(biography)) && (
              <ProfileBio>
                <ProfileBioTitle>Bio Overview</ProfileBioTitle>
                <ProfileBioText as={!empty(biography) && 'div'}>
                  <JSXWithDynamicLinks
                    HTML={attorneyBiography?.miniBio || biography}
                  />
                </ProfileBioText>
              </ProfileBio>
            )}

            <ProfileBioListItems>
              {!empty(education) && (
                <ProfileBioList title="Education" content={education} />
              )}
              {!empty(barAdmissions) && (
                <ProfileBioList title="Admissions" content={barAdmissions} />
              )}
              {!empty(affiliations) && (
                <ProfileBioList title="Affiliations" content={affiliations} />
              )}
              {!empty(additionalInfo)
                && additionalInfo.map(
                  (item) => !empty(item?.content) && (
                  <ProfileBioList
                    key={`${item?.title}-additional-info`}
                    title={item?.title}
                    content={item?.content}
                  />
                  ),
                )}
            </ProfileBioListItems>
          </ProfileHeaderRight>
        </ProfileHeaderHolder>
      </ContainerDefault>
    </ProfileHeaderSection>
  );
};

export default ProfileHeader;
