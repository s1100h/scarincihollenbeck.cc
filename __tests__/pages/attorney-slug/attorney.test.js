import {
  fireEvent,
  render,
  screen,
  waitFor,
  within,
} from '@testing-library/react';
import ProfileContacts from 'components/molecules/attorney/ProfileContacts';
import ProfileAccordion from 'components/organisms/attorney/ProfileAccordion';
import ProfileHeader from 'components/organisms/attorney/ProfileHeader';
import ReduxProvider from 'hoks/reduxTestHoc';
import ApolloWrapper from 'layouts/ApolloWrapper';
import { useRouter } from 'next/router';
import renderer from 'react-test-renderer';

jest.mock('next/router', () => ({
  useRouter: jest.fn(),
}));

useRouter.mockReturnValue({
  pathname: '/attorneys/[slug]',
  route: '/attorneys/[slug]',
  query: { slug: 'kaylin-n-olsen' },
  asPath: '/attorneys/kaylin-n-olsen',
});

jest.mock('next/image', () => ({
  __esModule: true,

  default: ({
    blurDataURL, priority, fill, ...props
  }) => (
    // eslint-disable-next-line @next/next/no-img-element, jsx-a11y/alt-text
    <img {...props} />
  ),
}));

describe('Attorney page', () => {
  it('Profile header component render', () => {
    const props = {
      name: 'Ronald S. Bienstock',
      profileImage: 'https://dummyimage.com/500x500/fff/aaa',
      contact: {
        vizibility: 'https://viz.me/Kolsen',
        pdf: 'https://res.cloudinary.com/scarinci-hollenbeck/wp.scarincihollenbeck/Olsen.pdf?_i=AA',
      },
    };
    render(
      <ReduxProvider>
        <ProfileHeader {...props} />
      </ReduxProvider>,
    );

    const profileHeaderContainer = screen.getByTestId('profile-header');
    expect(profileHeaderContainer).toBeInTheDocument();

    const titleProfile = within(profileHeaderContainer).getByRole('heading', {
      level: 1,
    });
    expect(titleProfile).toBeInTheDocument();
    expect(titleProfile.tagName).toBe('H1');
    expect(titleProfile.textContent).toBe(props?.name);

    const images = within(profileHeaderContainer).getAllByRole('img');
    const imageProfile = images.find((img) => img.alt === props?.name);
    expect(imageProfile).toBeInTheDocument();
    expect(imageProfile).toHaveAttribute(
      'src',
      expect.stringContaining(props?.profileImage),
    );

    const contactButton = within(profileHeaderContainer).getByRole('button', {
      name: /Contact now/i,
    });
    expect(contactButton).toBeInTheDocument();

    fireEvent.click(contactButton);

    const modal = profileHeaderContainer.querySelector('.modal-open');
    expect(modal).toBeInTheDocument();

    const printBioButton = within(profileHeaderContainer).getByRole('link', {
      name: /Print Bio/i,
    });

    expect(printBioButton).toHaveAttribute('href', props?.contact?.pdf);

    const businessCardButton = within(profileHeaderContainer).getByRole(
      'link',
      {
        name: /Business Card/i,
      },
    );
    expect(businessCardButton).toHaveAttribute(
      'href',
      props?.contact?.vizibility,
    );
  });

  it('Contacts snapshot', () => {
    const profileDetailsProps = {
      offices: [
        {
          link: '/location/little-falls',
          name: 'Little Falls',
          ID: 'cG9zdDoyOTQzNg==',
        },
      ],
      fax: '201-896-8660',
      contact: {
        phoneNumber: '201-896-7029 ',
        email: 'kolsen@sh-law.com',
        fax: '201-896-8660',
        vizibility: 'https://viz.me/Kolsen',
        pdf: 'https://res.cloudinary.com/scarinci-hollenbeck/wp.scarincihollenbeck/Olsen.pdf?_i=AA',
        socialMediaLinks: [
          {
            url: 'https://www.linkedin.com/company/scarinci-hollenbeck-llc/',
            channel: 'LinkedIn',
          },
        ],
      },
      linkedIn: {
        url: 'https://www.linkedin.com/company/scarinci-hollenbeck-llc/',
        channel: 'LinkedIn',
      },
    };
    const tree = renderer
      .create(<ProfileContacts {...profileDetailsProps} />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('Profile accordion component render', async () => {
    const props = {
      clients: [
        {
          clientImage: {
            sourceUrl:
              'https://res.cloudinary.com/scarinci-hollenbeck/images/f_auto,q_auto/v1642542645/wp.scarincihollenbeck/PEAVEY/PEAVEY.jpg?_i=AA',
            altText: 'PEAVEY',
            mediaDetails: { height: 300, width: 300 },
          },
          clientLink: null,
          clientTitle: 'peavey',
        },
        {
          clientImage: {
            sourceUrl:
              'https://res.cloudinary.com/scarinci-hollenbeck/images/f_auto,q_auto/v1642542638/wp.scarincihollenbeck/PHANTOM/PHANTOM.jpg?_i=AA',
            altText: 'PHANTOM',
            mediaDetails: { height: 300, width: 300 },
          },
          clientLink: null,
          clientTitle: 'phantom',
        },
      ],
    };
    render(
      <ApolloWrapper>
        <ProfileAccordion {...props} />
      </ApolloWrapper>,
    );

    await waitFor(() => {
      const accordionContainer = screen.getByTestId('profile-accordion');
      expect(accordionContainer).toBeInTheDocument();

      // Test clients section in accordion
      if (props?.clients?.length > 0) {
        const clientsAccordionItem = within(accordionContainer).getByTestId(
          'profile-clients-list',
        );
        expect(clientsAccordionItem).toBeInTheDocument();

        const titles = within(clientsAccordionItem).getAllByRole('heading', {
          level: 3,
        });

        const sliderTitle = titles.find(
          (title) => title.textContent === 'Clients',
        );
        expect(sliderTitle).toBeInTheDocument();

        const sliderContainer = clientsAccordionItem.querySelector('.slider-container');
        const clientSlides = within(sliderContainer).getAllByRole('img');
        expect(clientSlides).toHaveLength(props.clients.length);

        const clientsAccordionButton = within(clientsAccordionItem).getByRole(
          'button',
          { name: /Clients List/i },
        );

        expect(clientsAccordionButton).toBeInTheDocument();

        fireEvent.click(clientsAccordionButton);

        const clientsAccordionBody = clientsAccordionItem.querySelector('.accordion-body');
        const clientsList = within(clientsAccordionBody).getAllByRole('listitem');
        expect(clientsList).toHaveLength(props.clients.length);
      }

      const disclaimer = within(accordionContainer).getByText(
        /No Aspect of the advertisement has been approved by the Supreme Court. Results may vary depending on your particular facts and legal circumstances./i,
      );
      expect(disclaimer).toBeInTheDocument();
    });
  });
});
