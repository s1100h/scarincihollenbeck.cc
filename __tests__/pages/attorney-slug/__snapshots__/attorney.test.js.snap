// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attorney page Contacts snapshot 1`] = `
<address
  className="sc-132ec2da-12 exucvO"
>
  <p
    className="contacts-title"
  >
    Contacts
  </p>
  <ul
    className="contacts-list"
  >
    <li
      className="contacts-item"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          {
            "color": undefined,
          }
        }
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M12 2C8.13 2 5 5.13 5 9c0 1.74.5 3.37 1.41 4.84.95 1.54 2.2 2.86 3.16 4.4.47.75.81 1.45 1.17 2.26.26.55.47 1.5 1.26 1.5s1-.95 1.25-1.5c.37-.81.7-1.51 1.17-2.26.96-1.53 2.21-2.85 3.16-4.4C18.5 12.37 19 10.74 19 9c0-3.87-3.13-7-7-7zm0 9.75a2.5 2.5 0 010-5 2.5 2.5 0 010 5z"
        />
      </svg>
      <a
        className="contacts-link"
        href="/location/little-falls"
        onClick={[Function]}
        onMouseEnter={[Function]}
        onTouchStart={[Function]}
      >
        Little Falls
      </a>
    </li>
    <li
      className="contacts-item"
    >
      <a
        className="contacts-link"
        href="tel:201-896-7029 "
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            {
              "color": undefined,
            }
          }
          viewBox="0 0 24 24"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M0 0h24v24H0z"
            fill="none"
          />
          <path
            d="M6.62 10.79c1.44 2.83 3.76 5.14 6.59 6.59l2.2-2.2c.27-.27.67-.36 1.02-.24 1.12.37 2.33.57 3.57.57.55 0 1 .45 1 1V20c0 .55-.45 1-1 1-9.39 0-17-7.61-17-17 0-.55.45-1 1-1h3.5c.55 0 1 .45 1 1 0 1.25.2 2.45.57 3.57.11.35.03.74-.25 1.02l-2.2 2.2z"
          />
        </svg>
        <span>
          201-896-7029 
        </span>
      </a>
    </li>
    <li
      className="contacts-item"
    >
      <svg
        fill="currentColor"
        height="1em"
        stroke="currentColor"
        strokeWidth="0"
        style={
          {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M400 96V56a8 8 0 00-8-8H120a8 8 0 00-8 8v40"
        />
        <rect
          fill="none"
          height="160"
          rx="4"
          ry="4"
          width="208"
          x="152"
          y="264"
        />
        <rect
          fill="none"
          height="160"
          rx="4"
          ry="4"
          width="208"
          x="152"
          y="264"
        />
        <path
          d="M408 112H104a56 56 0 00-56 56v208a8 8 0 008 8h56v72a8 8 0 008 8h272a8 8 0 008-8v-72h56a8 8 0 008-8V168a56 56 0 00-56-56zm-48 308a4 4 0 01-4 4H156a4 4 0 01-4-4V268a4 4 0 014-4h200a4 4 0 014 4zm34-212.08a24 24 0 1122-22 24 24 0 01-22 22z"
        />
      </svg>
      201-896-8660
    </li>
    <li
      className="contacts-item"
    >
      <a
        className="contacts-link"
        href="mailto:kolsen@sh-law.com"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            {
              "color": undefined,
            }
          }
          viewBox="0 0 16 16"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M.05 3.555A2 2 0 0 1 2 2h12a2 2 0 0 1 1.95 1.555L8 8.414.05 3.555ZM0 4.697v7.104l5.803-3.558L0 4.697ZM6.761 8.83l-6.57 4.027A2 2 0 0 0 2 14h12a2 2 0 0 0 1.808-1.144l-6.57-4.027L8 9.586l-1.239-.757Zm3.436-.586L16 11.801V4.697l-5.803 3.546Z"
          />
        </svg>
        <span>
          kolsen@sh-law.com
        </span>
      </a>
    </li>
    <li
      className="contacts-item"
    >
      <a
        className="contacts-link"
        href="https://www.linkedin.com/company/scarinci-hollenbeck-llc/"
        rel="noreferrer"
        target="_blank"
      >
        <svg
          fill="currentColor"
          height="1em"
          stroke="currentColor"
          strokeWidth="0"
          style={
            {
              "color": undefined,
            }
          }
          viewBox="0 0 448 512"
          width="1em"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
          />
        </svg>
        <span>
          LinkedIn
        </span>
      </a>
    </li>
  </ul>
</address>
`;
